launcher/InstanceImportTask.cpp

```diff
@@ -55,11 +55,9 @@
 
 #include <quazip/quazipdir.h>
 
-InstanceImportTask::InstanceImportTask(const QUrl sourceUrl, QWidget* parent)
-{
-    m_sourceUrl = sourceUrl;
-    m_parent = parent;
-}
+InstanceImportTask::InstanceImportTask(const QUrl sourceUrl, QWidget* parent, QMap<QString, QString> extra_info)
+    : m_sourceUrl(sourceUrl), m_extra_info(std::move(extra_info)), m_parent(parent)
+{}
```

Comment:
What does `std::move` do here? Wouldn't we need to `std::move` while calling this constructor?
(`InstanceImportTask(coolUrl, coolParent, std::move(coolExtraInfo))`)